### ISSUE TITLE
Linting Issue

### ISSUE THREAD

**Issue by MannuVilasara** [EntySec/SeaShell#32]
**What is your question?**
So I recenty runned the pylint on the codebase and the results aren't that good. So is there any need to fix linting? also a workflow can be created to check wether the project really gets built, linting, formatting etc.

**Additional context**
![image](https://github.com/EntySec/SeaShell/assets/117009138/214c8f16-e977-41ff-b786-8218dd3cb07b)

**Any suggestions for the docs?**
can work on fixing the linting but will be a commit (probably gonna sqash commits while merging here).

**Comment by enty8080** [comment]
@MannuVilasara Do we need to have 10/10? What's the point of linting? The code is formatted following the PEP8 guidelines, I am not aware of other standards.

**Comment by MannuVilasara** [comment]
> @MannuVilasara Do we need to have 10/10? What's the point of linting? The code is formatted following the PEP8 guidelines, I am not aware of other standards.

Yea not really a need but just a suggestion. Can be used while creating workflows

**Comment by enty8080** [comment]
@MannuVilasara What command did you run in pylint?

My rating is `8.94/10`. Command: `pylint seashell`

**Comment by enty8080** [comment]
I guess you need to run `pylint seashell` to check whole module.

**Comment by MannuVilasara** [comment]
Won't it just check for just the seashell file present in bin

**Comment by enty8080** [comment]
No, because you are running pylint on a seashell directory (from the root of this repository). It checks for all files located in seashell

**Comment by MannuVilasara** [comment]
> No, because you are running pylint on a seashell directory (from the root of this repository). It checks for all files located in seashell

Yea it seems like it. Got the result to be 7.9 now.

**Comment by enty8080** [comment]
IDK why did you get different result. What is your python version? I guess this might be the reason (mine is 12)
